code/modules/projectiles/magazines/misc.dm

```diff
@@ -51,7 +51,7 @@
 
 /obj/item/ammo_magazine/packet/t25
 	name = "box of 10x26mm (T-25)"
-	desc = "A box containing 320 rounds of 10x26mm caseless tuned for a T-25 smartrifle.."
+	desc = "A box containing 320 rounds of 10x26mm caseless tuned for a T-25 smartrifle. It cannot be used for T-29 drums!"
```

Comment:
Does this really need to be said?
It's sounds like we're saying 9mm wont work with 7.62 ammo, which feels very redundant.

Reply:
True, it is redundant for players that know better, but for players that know both T-25 mag and T-29 drums have the same caliber name, 10x26mm, & do not that you cannot use box to refill drums with this ammo box, they may operate on the presumption they can refill drum with ammo box.

It's likely that this population is low in frequency and they just need to learn, though I do feel compel to add this to description.